### DISCUSSION TITLE
Space/Celestial%20Bodies/How%20to%20destroy%20a%20Black%20Hole

### DISCUSSION THREAD

**Discussion by giscus** [byanko55/jekyll-theme-satellite#53] [Q&A]
# Space/Celestial%20Bodies/How%20to%20destroy%20a%20Black%20Hole

you cant

https://mikoinspaceblog.github.io/Space/Celestial%20Bodies/How%20to%20destroy%20a%20Black%20Hole.html

**Comment by MikoInSpace** [comment]
Instructions unclear

**Reply by Za9-118** [reply]
mhm

**Comment by ElectroBoy404NotFound** [comment]
So.... Doesn't that mean that black holes are impossible to destroy?

**Reply by ElectroBoy404NotFound** [reply]
Or am I missing something?

**Comment by MikoInSpace** [comment]
basically, yeah

Am Di., 25. Juni 2024 um 13:22 Uhr schrieb CatDev ***@***.***
